src/core_plugins/kibana/public/management/sections/indices/index_pattern_list/index_pattern_list.js

```diff
@@ -17,29 +17,20 @@
  * under the License.
  */
 
-import React, { Component, Fragment } from 'react';
-import PropTypes from 'prop-types';
+import React, { Fragment } from 'react';
 
 import { Header } from './components/header';
 import { List } from './components/list';
 
-export class IndexPatternList extends Component {
-  static propTypes = {
```

Comment:
Does it make sense to keep the propTypes around?

```js
IndexPatternList.propTypes = {
  indexPatternCreationOptions: PropTypes.array.isRequired,
  defaultIndex: PropTypes.string,
  indexPatterns: PropTypes.array.isRequired,
};
```

Reply:
we could, but the propTypes are enforced by components that this component renders so I don't think it's necessary